src/reactor/mod.rs

```diff
@@ -479,7 +487,7 @@ impl Remote {
     pub fn handle(&self) -> Option<Handle> {
```

Comment:
This can be changed to always return `Handle` I think (instead of `Option`? I don't really remember why it returns option).